server/controllers/notification.js

```diff
@@ -0,0 +1,60 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+exports.createNotification = asyncHandler(async (req, res, next) => {
+    try {
```

Comment:
We don't need a try/catch block as the `asyncHandler` will handle promise rejections and forward it to our error middleware.

Ditto where applicable!